src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import { Routes, Route } from 'react-router-dom';
+import Books from './Books';
+import Navbar from './Navbar';
+import '../styles/App.module.scss';
+
+const BookstoreApp = () => (
+  <Routes>
+    <Route path="/" element={<Navbar />}>
+      <Route index element={<Books />} />
```

Comment:
- [ ] Great job adding route to the books component. Kindly add a route to the categories element. Also, add the categories component as demanded by project requirements.
![image](https://github.com/Bifabig/bookstore-react/assets/94551484/760dd9ec-f546-4ba2-832e-82cbcd355319)

Reply:
Hi @Kamba56, Kindly review it again... I've made the necessary changes.

Reply:
This change has been done well